components/eam/cime_config/config_component.xml

```diff
@@ -62,7 +62,7 @@
       <value compset="_EAM%SCM_"       >&eam_phys_defaults; &eam_chem_defaults; -scam</value>
 
       <!-- idealized configurations -->
-      <value compset="_EAM%AQP_"       >&eam_phys_defaults; &eam_chem_defaults; -aquaplanet</value>
+      <value compset="_EAM%AQP_"       >-nlev 72 -clubb_sgs -microphys mg2 -chem linoz_mam4_resus_mom_soag -rain_evap_to_coarse_aero -aquaplanet</value>
```

Comment:
Why are we reverting to the v2 config for aquaplanet?

Reply:
Good point, @whannah1. FAQP compset appears to use some settings available with MG2 (e.g., micro_do_nccons, micro_do_nicons). I think there must be a reason doing that. Until having equivalent implementation in P3, and it should be more appropriate to continue using v2 physics for testing purpose. A small bobus with this approach, the FAQP test with this PR is BFB with master baseline.

Reply:
Sounds good for RCE to have similar physics. The PR is merged. Will keep in mind and do it in alter PR, one way or another to both FAQP and RCE regarding physics option.